docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss

```diff
@@ -38,7 +38,7 @@
     @include grid-media-min('lg') {
       .hb-three-column-w-image--one-sidebar &,
       .hb-three-column-w-image--two-sidebar & {
-        width: 100%;
+        width: calc(100% - #{$hb-sidebar-width} - #{hb-calculate-rems($hb-gutter-width)});
```

Comment:
lol good catch

Reply:
hmm but if there's two sidebars, I wonder if the logic is still messed up

Reply:
I think I checked and it looks ok on different breakpoints, but it'd be great if you could check too!

Reply:
I double checked. Looks good to me! :eyes: